github-action/main.go

```diff
@@ -97,6 +98,26 @@ func getApplicationIds(qoveryAPIClient pkg.QoveryAPIClient, envId string, id *st
 	return "", errors.New("'app-ids' or 'app-names' property must be defined")
 }
 
+func getContainerIds(qoveryAPIClient pkg.QoveryAPIClient, envId string, id *string, name *string) (string, error) {
+	if id != nil && *id != "" {
+		return sanitizeInputIDsList(*id), nil
+	}
+
+	if name != nil && *name != "" {
+		var ids []string
+		for _, sName := range strings.Split(*name, ",") {
```

Comment:
Nit: I know other similar functions doesn't have this in place but I would make sure to trim names / IDs generally speaking.
We have been hit by a similar case in last couple weeks, and might happen here as well.

CF PR fixing it => https://github.com/Qovery/qovery-github-actions/pull/11

Reply:
Updated :)